### DISCUSSION TITLE
Custom UI [Sep 01]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#3737] [Discord Threads]
> 🙋‍♂️ **ccoli**

**Comment by o1lab** [comment]
> 🙋‍♂️ **wkw**  
#### have you defined `NC_DB` when u start the app  
#  
> 🙋‍♂️ **wkw**  
#### those meta data is stored in `NC_DB` . if you don't specify it, then a local sqlite db will be created in your docker container. if you haven't mounted the volume, then it will be lost whenever u restart it  
#  
> 🙋‍♂️ **ccoli**  
#### ok thanks very much. u mean without volume, it will be lost anyway even though with NC_DB right?   
####    
#### thank u very much  
#  
> 🙋‍♂️ **wkw**  
#### u can check out our docker examples   
####    
#### https://github.com/nocodb/nocodb#docker  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1012252016266977340)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
